File: src/NzbDrone.Core/MetadataSource/Tmdb/TmdbEpisodeGroup.cs
namespace NzbDrone.Core.MetadataSource.Tmdb
{
    public class TmdbEpisodeGroup
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public TmdbEpisodeGroupType Type { get; set; }
        public int EpisodeCount { get; set; }
        public int GroupCount { get; set; }
    }

    // https://developer.themoviedb.org/reference/tv-series-episode-groups
    public enum TmdbEpisodeGroupType
    {
        OriginalAirDate = 1,
        Absolute = 2,
        Dvd = 3,
        Digital = 4,
        StoryArc = 5,
        Production = 6,
        Tv = 7
    }
}

File: frontend/src/Series/TmdbEpisodeGroup.ts
export type TmdbEpisodeGroupType =
  | 'originalAirDate'
  | 'absolute'
  | 'dvd'
  | 'digital'
  | 'storyArc'
  | 'production'
  | 'tv';

interface TmdbEpisodeGroup {
  id: string;
  name: string;
  description: string;
  type: TmdbEpisodeGroupType;
  episodeCount: number;
  groupCount: number;
}

export default TmdbEpisodeGroup;
